### DISCUSSION TITLE
Welcome to AutoFixture Discussions!

### DISCUSSION THREAD

**Discussion by aivascu** [AutoFixture/AutoFixture#1216] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Kralizek** [comment]
Hi everybody.

I've been using AutoFixture for quite some time and I have been pestering @zvirja and @aivascu on potential features and improvements for quite some time.

I've also added some minor concrete contribution to the project and I can say I really love how it makes writing tests so quick and easy.

As a (former) manager of a team of 25 developers, I could see the difficulties neophytes could encounter when using this library and I am continuously pushing for a simplification and clarification of its API, at least on the boundaries.

I noticed that many of my "issues" ended up here on the discussion board and I hope that's a good thing! :)

**Reply by aivascu** [reply]
@Kralizek your involvement with AutoFixture is greatly appreciated! Please do continue to pester us! 😄
We need more active community members like yourself, to help AutoFixture evolve! 🚀

PS: I am fairly sure this is a good thing! We will post an update to Twitter soon explaining this change.

**Comment by aivascu** [comment]
Hello everyone!

I'm Andrei, and I'm a maintainer of AutoFixture. I've joined the team in June 2020, that making me the latest member to join. 
Since then, I've been slowly learning the ropes of this project and trying to improve it.

My goal was and still is to revitalize the AutoFixture development as well as raise awareness and adoption of this amazing tool! 🚀 

You can find me on Twitter [@_aivascu](https://twitter.com/_aivascu), on [StackOverflow](https://stackoverflow.com/questions/tagged/autofixture) answering questions about AutoFixture and now here in the [Discussions](https://github.com/AutoFixture/AutoFixture/discussions) section. 🙂
